deps/build-deps-linux.sh

```diff
@@ -138,10 +147,11 @@ rm -rf lcms
 rm -rf ffmpeg
 rm -rf libyuv
 rm -rf aom
+rm -rf dav1d
 rm -rf libavif
 
 if [ ! -d "$SRCDIR" ]; then
-    git clone --depth 1 --branch 1.4.1 https://github.com/discord/lilliput-dep-source "$SRCDIR"
+    git clone --depth 1 --branch skidder/add-libdav1d https://github.com/discord/lilliput-dep-source "$SRCDIR"
```

Comment:
iirc you want to:
1. keep this pointing to 1.4.1, ensure flag is off
2. update & bump lilliput-dep-source
3. bump here
4. flip flag